### PR TITLE
Add hours-minutes-converter extension

### PR DESCRIPTION
**Pull request by ItzOuluhh**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by ItzOuluhh** [comment]
A simple extentions for converting hours and minutes to decimals and vice versa.

File: extensions/hours-minutes-converter/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# hours:minutes converter Changelog
```
Comment (pernielsentikaer) [inline extensions/hours-minutes-converter/CHANGELOG.md:1]:
```suggestion
# Hundredths to Minutes Changelog
```

File: extensions/hours-minutes-converter/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Hours-Minutes to Decimal Converter for Raycast
```
Comment (pernielsentikaer) [inline extensions/hours-minutes-converter/README.md:1]:
```suggestion
# Hundredths to Minutes for Raycast
```

File: extensions/hours-minutes-converter/package.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "hours-minutes-converter",
+  "title": "Hours:Minutes Converter",
```
Comment (pernielsentikaer) [inline extensions/hours-minutes-converter/package.json:4]:
```suggestion
  "title": "Hundredths to Minutes",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
- I would (maybe) make it a `no-view` command instead, and just show a HUD (and maybe copy automatically)

**Comment by pernielsentikaer** [comment]
It's also possible directly in Raycast, maybe you can just use that 

image

**Comment by ItzOuluhh** [comment]
Does this mean my extension has been denied? Thanks for your reaction!

**Pull request closed without merging**